src/Npgsql.DependencyInjection/Npgsql.DependencyInjection.csproj

```diff
@@ -3,8 +3,8 @@
     <PropertyGroup>
         <Authors>Shay Rojansky</Authors>
         <!-- DbDataSource was introduced to .NET in net7.0. Before that Npgsql has its own built-in copy. -->
-        <TargetFrameworks>netstandard2.0;net7.0</TargetFrameworks>
-        <TargetFrameworks Condition="'$(DeveloperBuild)' == 'True'">net7.0</TargetFrameworks>
+        <TargetFrameworks Condition="'$(DeveloperBuild)' != 'True'">netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
This condition is a good idea - I've had "conflicts" between the two from time to time, thanks! But note that it needs to be added also to the other csprojs (e.g. Npgsql.GeoJSON.csproj)

Reply:
> But note that it needs to be added also to the other csprojs (e.g. Npgsql.GeoJSON.csproj)

No, you'r misunderstanding this.. We have to add it where the project file uses (or inherits from Directory.Build.props) a \<TargetFramework***s***> entry as default. If it uses a <TargetFramework> (without ***s***) entry (as it is the case in Npgsql.GeoJSON.csproj), we just override it in case of a developer build and everything works as intended.
The problem is, that we cannot override <TargetFramework***s***> with <TargetFramework> (without ***s***).

Reply:
Makes sense :+1: